core/tests/protractor_utils/forms.js

```diff
@@ -130,7 +133,8 @@ var ListEditor = function(elem) {
   // If objectType is not specified, this function returns nothing.
   var addItem = async function(objectType = null) {
     var listLength = await _getLength();
-    await elem.element(by.css('.protractor-test-add-list-entry')).click();
+    var addListButton = elem.element(by.css('.protractor-test-add-list-entry'));
+    await action.click('Test add List Entry Button', addListButton);
     if (objectType !== null) {
       return await getEditor(objectType)(
         elem.element(
```

Comment:
Could you use `action.click()` for the `.click()` below?

Reply:
updated!